rconweb/api/views.py

```diff
@@ -254,6 +255,16 @@ def players_history(request):
         "failed": failed
     })
 
+@csrf_exempt
+def get_instances(request):
+    instances = get_registered_instance()
+    return JsonResponse({
+        "result": instances,
+        "command": "get+instances",
```

Comment:
typo _ instead of +

Reply:
Ah ok, i just assumed that was your preferred formatting so i didn't look any deeper :D